app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/CheckboxInput/index.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
```

Comment:
as we are not using this component in more than one place
can we move this directly yo the if else conditions in app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/InputWidgets/index.tsx

Reply:
Done